src/project-management/project-management.ts

```diff
@@ -40,20 +40,23 @@ class ProjectManagementInternals implements FirebaseServiceInternalsInterface {
 }
 
 /**
- * ProjectManagement service bound to the provided app.
+ * The Firebase ProjectManagement service interface.
+ *
+ * Do not call this constructor directly. Instead, use
+ * [`admin.projectManagement()`](admin.projectManagement#projectManagement).
  */
 export class ProjectManagement implements FirebaseServiceInterface {
-
+  /** @internal */
```

Comment:
Let's not use `@internal` yet. Let things be exposed to the auto-generated typings for now. We can fix them in a future iteration.